src/Microsoft.AspNetCore.Rewrite/RewriteMiddleware.cs

```diff
@@ -77,10 +77,11 @@ public Task Invoke(HttpContext context)
                 rule.ApplyRule(rewriteContext);
                 var currentUrl = string.Empty;
 
-                if (rewriteContext.Logger?.IsEnabled(LogLevel.Debug) == true)
+                if (rewriteContext.Logger?.IsEnabled(LogLevel.Debug | LogLevel.Information) == true)
```

Comment:
Use `.IsEnabled(LogLevel.Debug) || logger.IsEnabled(LogLevel.Information)` instead. LogLevel isn't designed to be used as bit flags.